test/index.js

```diff
@@ -1831,7 +1840,16 @@ describe('def', function() {
 
     //  unnest :: Array (Array a) -> Array a
     var unnest =
-    def('unnest', {}, [$.Array($.Array(a)), $.Array(a)], R.unnest);
+    def('unnest',
+        {},
+        [$.Array($.Array(a)), $.Array(a)],
+        function(xss) {
+          var xs = [];
+          for (var idx = 0; idx < xss.length; idx += 1) {
+            Array.prototype.push.apply(xs, xss[idx]);
+          }
+          return xs;
+        });
```

Comment:
Indeed! This led me to realize that several lambdas in __test/index.js__ could be replaced with `Z.concat`, `Z.map`, or `Z.filterM`. This is nice as it keeps the tests focused on the *types* of the functions.